lib/blueprinter/base.rb

```diff
@@ -68,24 +128,86 @@ def self.prepare(object, view:)
       end
     end
 
+    # Specify one or more field/method names to be included for serialization.
+    # Takes at least one field or method names.
+    #
+    # @param method [Symbol] the field or method name you want to include for
+    #   serialization.
+    #
+    # @example Specifying a user's first_name and last_name to be serialized.
+    #   class UserBlueprint < Blueprinter::Base
+    #     fields :first_name, :last_name
+    #     # other code
+    #   end
+    #
+    # @return [Array<Symbol>] an array of field names
     def self.fields(*field_names)
       field_names.each do |field_name|
         current_view << Field.new(field_name, field_name, PublicSendSerializer)
       end
     end
 
+    # @api private
     def self.associations(view_name = :default)
       view_collection.fields_for(view_name).select { |f| f.options[:association] }
     end
 
+    # Specify another view that should be mixed into the current view.
+    #
+    # @param view_name [Symbol] the view to mix into the current view.
+    #
+    # @example Including a normal view into an extended view.
+    #   class UserBlueprint < Blueprinter::Base
+    #     # other code...
+    #     view :normal do
+    #       # some fields set here
+    #     end
+    #     view :extended do
+    #       include_view :normal # include fields specified from above.
+    #       field :description
+    #     end
+    #   end
```

Comment:
We should probably augment this example to provide even more clarity - add maybe a `field :name` in the `:normal` view and then show how the output includes both `[:name, :description]`